app/views/answers/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row static-width main-content-container">
+
+  <div class="col-md-7 col-md-offset-1">
+    <%= render partial: 'layouts/errors', locals: { element: @answer } %>
+    
+    <ul class="no-list-style no-padding">
+      <%= form_for [@question, @answer], url: answer_path do |t| %>
```

Comment:
Your answer update route is not nested under question, so you don't need @question were

`form_for @answer` should just work

(Or you could remove the shallow as we discussed.)